apps/framework-cli/src/framework/typescript/generator.rs

```diff
@@ -305,32 +277,24 @@ pub fn std_table_to_typescript_interface(
 }
 
 fn collect_ts_objects(
-    project: &Project,
     version: &str,
     framework_objects: &SchemaVersion,
 ) -> Result<Vec<TypescriptObjects>, TypescriptGeneratorError> {
     let mut ts_objects: Vec<TypescriptObjects> = Vec::new();
 
     for model in framework_objects.get_all_models() {
         let interface = std_table_to_typescript_interface(model.to_table(version), &model.name)?;
-
-        let send_function = SendFunction::new(
-            interface.clone(),
-            format!("{}/{}", project.http_server_config.host, "api"),
-            model.name.clone(),
-        );
-
-        ts_objects.push(TypescriptObjects::new(interface, send_function));
+        ts_objects.push(TypescriptObjects::new(interface));
     }
 
     Ok(ts_objects)
 }
 
-fn collect_enums(framework_objects: &SchemaVersion) -> Vec<TSEnum> {
-    let mut enums: Vec<TSEnum> = Vec::new();
+fn collect_enums(framework_objects: &SchemaVersion) -> HashSet<TSEnum> {
+    let mut enums: HashSet<TSEnum> = HashSet::new();
```

Comment:
Does removing order here end up impacting languages like python where order of declarations can make an impact on number of tries that it takes to accomplish a task? i.e. if you have nested structures and the nested structure isn't declared ahead of the other one, you can cause errors.

Reply:
The above order is only impacting order in the import statements. Not structural so it should be fine.